devices/Tm1637/Tm1637.nfproj

```diff
@@ -49,17 +53,17 @@
   </ItemGroup>
   <Import Project="$(NanoFrameworkProjectSystemPath)NFProjectSystem.CSharp.targets" Condition="Exists('$(NanoFrameworkProjectSystemPath)NFProjectSystem.CSharp.targets')" />
   <Import Project="..\..\src\Stopwatch\Stopwatch.projitems" Label="Shared" />
```

Comment:
For Stopwatch, please use the nuget as it's available now rather than the link to the shared project.